arch/arm64/boot/dts/qcom/sdm660-xiaomi-clover-common.dtsi

```diff
@@ -174,6 +185,14 @@
 	status = "okay";
 };
 
+&pm660l_gpios {
+	lcd_pwr_en_default: lcd-pwr-en-default-state {
+		pins = "gpio5";
+		function = "gpio";
+		bias-disable;
```

Comment:
It's hard to find documentation sometimes, but for pmic GPIOs valid function names are documented here https://elixir.bootlin.com/linux/v6.11/source/Documentation/devicetree/bindings/pinctrl/qcom,pmic-gpio.yaml#L500 

and also as CI check-dtbs says
```
gpio' is not one of ['normal', 'paired', 'func1', 'func2', 'dtest1', 'dtest2', 'dtest3', 'dtest4', 'func3', 'func4']
	from schema $id: http://devicetree.org/schemas/mfd/qcom,spmi-pmic.yaml#
/home/runner/work/linux/linux/build-dtbs-check/arch/arm64/boot/dts/qcom/sdm660-xiaomi-clover-plus.dtb: gpio@c000: lcd-pwr-en-default-state: 'oneOf' conditional failed, one must be fixed:
	'bias-disable', 'function', 'pins' do not match any of the regexes: '(pinconf|-pins)$', 'pinctrl-[0-9]+'
	'gpio' is not one of ['normal', 'paired', 'func1', 'func2', 'dtest1', 'dtest2', 'dtest3', 'dtest4', 'func3', 'func4']
	from schema $id: http://devicetree.org/schemas/pinctrl/qcom,pmic-gpio.yaml#
```
so at least "function" should be changed to "normal" I guess

Valid names for functions are also here: https://elixir.bootlin.com/linux/v6.11/source/include/dt-bindings/pinctrl/qcom,pmic-gpio.h#L98